src/quota_aggregator_impl_test.cc

```diff
@@ -588,38 +628,117 @@ TEST_F(QuotaAggregatorImplTest, TestCacheRefreshQuotaModeBestEffort) {
             QuotaOperation_QuotaMode::QuotaOperation_QuotaMode_BEST_EFFORT);
 }
 
-TEST_F(QuotaAggregatorImplTest, TestCacheRefreshQuotaModeNormal) {
-  std::set<std::pair<std::string, int>> quota_metrics;
-  std::set<std::pair<std::string, int>> expected_costs;
-
+TEST_F(QuotaAggregatorImplTest, TestCacheRefreshQuotaCallForNegative) {
   AllocateQuotaResponse response;
 
   // trigger initial allocation
-  EXPECT_ERROR_CODE(StatusCode::kOk, aggregator_->Quota(request1_, &response));
-  // check refresh
+  EXPECT_OK(aggregator_->Quota(request1_, &response));
+  // quota_call count should be 1
   EXPECT_EQ(flushed_.size(), 1);
```

Comment:
What is stored in flushed_? Is it keeping track of the number of quota calls to the transport?

Reply:
Yes,  variable name is confusing. But I don't want to make too much changes in this PR